### PR TITLE
Added PhpUnitConstructFixer

### PR DESCRIPTION
**Pull request by keradus**
New fixer requested in #1208
Need to resolve TODO after merge of #1279

### CODE REVIEW

**Comment by keradus** [comment]
I think that this and #1279 fixer could be the last that come into v1.10 release.
If we decided to merge it.

**Comment by Anahkiasen** [comment]
Aren't there a lot of other constructs that could be fixed in this way?

| Before                                           | After                                  |
| ---                                              | ---                                    |
| `$this->assertEquals(5, count($foobar))`         | `$this->assertCount(5, $foobar)`       |
| `$this->assertTrue(empty($foobar))`              | `$this->assertEmpty($foobar)`          |
| `$this->assertTrue(file_exists($foobar))`        | `$this->assertFileExists($foobar)`     |
| `$this->assertTrue(preg_match($regex, $foobar))` | `$this->assertRegExp($regex, $foobar)` |

etc.

**Comment by keradus** [comment]
Yes and no.
For now the proposed fixer fixes most common misuses when there are no extra logic in passing argument.
When using `assertTrue` one can imitate every other exceptions via `assertTrue(magicFnc($foo))`

File: Symfony/CS/Fixer/Contrib/PhpUnitConstructFixer.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        $sequenceIndexes[4] = $tokens->getNextMeaningfulToken($sequenceIndexes[3]);
+        $firstParameterToken = $tokens[$sequenceIndexes[4]];
+
+        if (!$firstParameterToken->isNativeConstant()) {
+            return;
+        }
+
+        $sequenceIndexes[5] = $tokens->getNextNonWhitespace($sequenceIndexes[4]);
+
+        $tokens[$sequenceIndexes[2]]->setContent($map[$firstParameterToken->getContent()]);
```
Comment (stof) [inline Symfony/CS/Fixer/Contrib/PhpUnitConstructFixer.php]: what if it is not in the map ?

Comment (keradus) [inline Symfony/CS/Fixer/Contrib/PhpUnitConstructFixer.php]: then it won't pass `isNativeConstant` test and don't reach this line

**Pull request closed without merging**